components/brave_wallet/browser/ethereum_provider_impl.cc

```diff
@@ -1069,11 +950,8 @@ void EthereumProviderImpl::CommonRequestOrSendAsync(
   }
 
   if (method == kEthAccounts || method == kEthCoinbase) {
-    GetAllowedAccounts(
-        false,
-        base::BindOnce(&EthereumProviderImpl::OnContinueGetAllowedAccounts,
-                       weak_factory_.GetWeakPtr(), std::move(callback),
-                       std::move(id), method, delegate_->GetOrigin()));
+    OnContinueGetAllowedAccounts(std::move(callback), std::move(id), method,
```

Comment:
`OnContinueGetAllowedAccounts` actually don't know what proper name that should have.